packages/web-components/src/components/leaving-ibm/leaving-ibm.scss

```diff
@@ -30,19 +30,23 @@
 :host(#{$dds-prefix}-leaving-ibm-modal-body) {
   @extend :host(#{$prefix}-modal-body);
 
+  #{$prefix}-link {
+    @include carbon--type-style('body-long-02', true);
+  }
+
   padding-right: 20%;
 
   ::slotted(slot[name='supplemental']) {
     display: block;
     margin-top: $spacing-07;
     color: $text-02;
-    @include carbon--type-style('label-01');
+    @include carbon--type-style('label-02');
   }
 }
 
 :host(#{$dds-prefix}-leaving-ibm-modal-supplemental) {
   display: block;
   margin-top: $spacing-07;
   color: $text-02;
-  @include carbon--type-style('label-01');
+  @include carbon--type-style('label-02');
 }
```

Comment:
Looks like this was here previously, but any idea why these values are duplicated under slightly different selectors? Isn't there only one label?

Reply:
@photodow I have no idea why it was there but I tested it and the first one doesn't seem to be doing anything. I removed it.